database/sqlite/sqlite_aclk.c

```diff
@@ -0,0 +1,866 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "sqlite_functions.h"
+#include "sqlite_aclk.h"
+
+// TODO: To be added
+//#include "sqlite_aclk_chart.h"
+//#include "sqlite_aclk_alert.h"
+//#include "sqlite_aclk_node.h"
+
+#ifndef ACLK_NG
+#include "../../aclk/legacy/agent_cloud_link.h"
```

Comment:
the relative paths necessary?

Reply:
Not needed at all for this PR (will be removed)